examples/next/rpc-client.yml

```diff
@@ -0,0 +1,73 @@
+asyncapi: '2.0.0'
+id: 'urn:rpc:example:client'
+defaultContentType: application/json
+
+info:
+  title: RPC Client Example
+  description: This example demonstrates how to define an RPC client.
+  version: '1.0.0'
+
+servers:
+  - url: rabbitmq.example.org
+    scheme: amqp
+
+channels:
+  '{queue}':
+    parameters:
+      - name: queue
+        schema:
+          type: string
+          pattern: '^amq\\.gen\\-.+$'
+    protocolObjects:
+      amqp-0-9-1:
+        channelIsQueue: true
+        queue:
+          randomName: true
+          exclusive: true
+    subscribe:
+      operationId: receiveSumResult
+      correlationId:
+        location: $message.header#/correlation_id
+      protocolObjects:
+        amqp-0-9-1:
+          noAck: true
+          properties:
+            correlation_id:
+              type: string
```

Comment:
Please, ignore the "quality" of the `amqp-0-9-1` object here. The definition of a specific protocol object must be done separately. This example is just to illustrate the feature.